onnxruntime/test/python/transformers/test_flash_attn_cuda.py

```diff
@@ -47,6 +43,7 @@ class Config:
     num_heads = 0
     kv_num_heads = 0
     head_size = 0
+    ep = "CUDAExecutionProvider"
```

Comment:
update __repr__() function below to output current ep.

Reply:
Done